dstyle.dd

```diff
@@ -135,7 +135,22 @@ ubyte asciiChar;
 -------------------------------
     )
 
+    $(DT $(LNAME2 naming_udas, User-Defined Attributes))
+    $(DD For symbols that are only to be used as user-defined attributes the
+         names should be camelCased, so their first letter is lowercase. This
+         convention takes precedence over any previously mentioned conventions.
+         This matches conventions of the built in attributes like `@safe`,
+         `@nogc` and the special compiler recognized UDA `@selector`.
 
+-------------------------------
+struct Foo {} // this struct follows the regular naming conventions
+
+// this struct is only intended to by used as an UDA and therefore overrides the
```

Comment:
typo in the comment.

to by used as an -> to **be** used as an

Reply:
Fixed.